var/spack/repos/builtin/packages/qd/package.py

```diff
@@ -13,12 +13,14 @@ class Qd(AutotoolsPackage):
 
     homepage = "https://bitbucket.org/njet/qd-library/src/master/"
     git = "https://bitbucket.org/njet/qd-library.git"
+    url = "https://www.davidhbailey.com/dhbsoftware/qd-2.3.13.tar.gz"
 
     tags = ["hep"]
 
     license("BSD-3-Clause-LBNL")
-
-    version("2.3.13", commit="a57dde96b3255b80f7f39cd80217c213bf78d949")
+    version("2.3.24", sha256="a47b6c73f86e6421e86a883568dd08e299b20e36c11a99bdfbe50e01bde60e38")
+    version("2.3.23", sha256="b3eaf41ce413ec08f348ee73e606bd3ff9203e411c377c3c0467f89acf69ee26")
+    version("2.3.13", sha256="b3eaf41ce413ec08f348ee73e606bd3ff9203e411c377c3c0467f89acf69ee26")
```

Comment:
Identical checksum seems incorrect.

Reply:
I checked, and 2.3.13 and 2.3.23 have the same content.  It appears that they reset the version number.

Reply:
I added a comment to note that the identical sha256 was intentional.